internal/bminventory/inventory.go

```diff
@@ -4255,12 +4255,15 @@ func (b *bareMetalInventory) EnableHost(ctx context.Context, params installer.En
 func (b *bareMetalInventory) refreshHostAndClusterStatuses(
 	ctx context.Context,
 	eventName string,
-	hostID *strfmt.UUID,
+	host *common.Host,
 	clusterID *strfmt.UUID,
 	db *gorm.DB) (*common.Cluster, error) {
 
 	logger := logutil.FromContext(ctx, b.log)
 
+	hostID := host.ID
+	infraEnvID := host.InfraEnvID
```

Comment:
don't have that strong opinion about it, but instead of defining of a variable only for the sake of the defer, this can be set as a parameter to the defer function, then it will be relevant in the context in which it is relevant.

Reply:
done